### ISSUE TITLE
I would like a good recipe for rib-eye steak

### ISSUE THREAD

**Issue by nul800sebastiaan** [nul800sebastiaan/recipes#2]
What's really missing from this list is rib-eye steak. 

This one looks good but the beans are putting me off. I'm sure we can make a good variation on it though? 

https://www.jamieoliver.com/recipes/beef-recipes/epic-rib-eye-steak/

Is this along the lines of where we want to go with this cookbook?

**Comment by MikeUmbracoHQ** [comment]
I totally agree with you that a rib-eye steak is missing. I've looked at your suggestion and has committed a little different variant with chili, primarily due to the fact that this is the most popular recipe and the fact that it requires a dutch oven:)
